src/po/fi.po

```diff
@@ -588,20 +589,20 @@ msgid "unknown"
 msgstr "tuntematon"
 
 msgid "Greetings, Vim user!"
-msgstr "Tervehdys, Vimin käyttäjä."
```

Comment:
I've changed some exclamation marks to full stops in Finnish since it seems more idiomatic Finnish to me, we don't use exclamation marks as much as English but I'm ok either way.

Reply:
reverted them to as you had them. also changed E342 to follow that for consistency